client/src/management/owner.py

```
@@ -3,38 +3,45 @@
 from common.eth import EthereumClient
 from common.config import GasStrategy
 from common.logging import get_logger
+from models.execution_layer.model_registry import ModelValidationError
```

Flake8 (7.3.0)

[error] 'models.execution_layer.model_registry.ModelValidationError' imported but unused

(F401)